apps/style/rollups.scss

```diff
@@ -0,0 +1,2 @@
+@import "markdown";
+@import "pdf_printing";
```

Comment:
it's probably time to create our own subdirectory here. apps/style/curriculum, maybe?

Reply:
sorry for this curveball. this doesn't need to be done in this PR.

Reply:
I will follow up in another PR with this